static/js/pv_tree.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from "react";
+import hierarchy from "../../tools/pv_tree_generator/hierarchy_golden.json";
+//import hierarchy from "../data/hierarchy.json";
+
+interface NodePropType {
+  title: string;
+  selected: string;
+  expanded: string;
```

Comment:
do think selected and expanded can be removed? It's okay to do in another PR if so, and also updated the hierachy.json

Reply:
I'll do it in another PR and update the hierarchy.json.